crypto/src/test/kotlin/web5/sdk/crypto/Secp256k1Test.kt

```diff
@@ -53,6 +55,34 @@ class Secp256k1Test {
     val base64UrlEncodedSig2 = Convert(sig2).toBase64Url(padding = false)
 
     assertEquals(base64UrlEncodedSig1, base64UrlEncodedSig2)
+  }
+
+  @Test
+  fun fuzz() {
```

Comment:
```suggestion
  fun `can verify 10k random payloads`() {
```

or whatevs, not sure what the intent here is

Reply:
point here is to generate a sufficient number of signatures (aka 2 big ints) across a large enough surface area to suss out any edge cases there may be with signature verification

Reply:
cool with "pressure test signature verification" as the test name?

Reply:
yup 👍

Reply:
done - https://github.com/TBD54566975/web5-kt/commit/8049d78d49280d274a377d215016c9cff21d56f8